src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

Comment:
To ensure your CSS is robust and adaptable, avoid fixing the size of containers like `.stars__star`. If there is no explicit requirement to fix the size, consider using padding and allow the size to be determined by the content or context.